frontend/src/views/SynonymsPage.vue

```diff
@@ -107,7 +107,8 @@ const openArrItemList = async (app) => {
                     options = [...options, ...result.alternateTitles.map(({title}) => title)]
                 }
                 formModal.close();
-                const from = await dialogService.select(result.title, "Select a search Term", undefined, options);
+                let from = await dialogService.select(result.title, 'Select a search Term', undefined, options);
+                from = from.replaceAll(/[^a-zA-Z0-9\s.]/g, '');
```

Comment:
Found what Sonarr does: https://github.com/Sonarr/Sonarr/blob/94f64435f558acd9c3e476eb55909998e1290e02/src/NzbDrone.Core/IndexerSearch/Definitions/SearchCriteriaBase.cs#L28

This should allow for things like `-` and `'` in show titles, for example.